xfel/merging/application/modify/aux_cosym.py

```diff
@@ -24,6 +22,10 @@ def __init__(self, *args, **kwargs):
     self.output_dir = kwargs.pop('output_dir', None)
     super(CosymAnalysis, self).__init__(*args, **kwargs)
 
+  def run(self):
+    super(CosymAnalysis, self).run()
```

Comment:
For Python 3 you don't need to provide any arguments to `super()`, i.e. the following is equivalent (and similarly for the `super().__init__` call above which isn't currently touched by this PR):
```suggestion
    super().run()
```

Reply:
I think our syntax checker would complain about that.  We are still running a py2 syntax checker on cctbx_project.

Reply:
(Note: syntactically this is fine. It just doesn't work on Python 2. However, Python 2 will already have crashed out way before it reaches this line, since it won't be able to import anything from dials.)